src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -173,10 +202,20 @@ export default function SingleSelectableContent({
         <AutoSizer>
           {({ height, width }) => (
             <List
+              ref={listRef}
               height={height}
               itemCount={rows.length} // how many items we are going to render
               itemSize={convertRemToPixels(tableRowHeight[rowHeight])} // height of each row in pixel
               width={width}
+              onItemsRendered={({
+                visibleStartIndex,
+                visibleStopIndex,
+                overscanStopIndex,
+              }) => {
+                setHasScrollbar(
+                  visibleStopIndex - visibleStartIndex < overscanStopIndex,
+                );
+              }}
```

Comment:
I think this can be dropped.  If outerRef.offsetWidth == innerRef.offsetWidth then in TabelStyle:

```
width: calc(100% - ${props.scrollBarWidth> 0?props.scrollBarWidth-4:0}px )!important;// -4px for border 
```